src/zmqpp/socket_options.hpp

```diff
@@ -116,6 +116,7 @@ ZMQPP_COMPARABLE_ENUM socket_option {
 	heartbeat_timeout         = ZMQ_HEARTBEAT_TIMEOUT, /*< ZMPT heartbeat timeout - set only */
 	heartbeat_ttl             = ZMQ_HEARTBEAT_TTL, /*< ZMPT heartbeat interval - set only */
 	invert_matching           = ZMQ_INVERT_MATCHING, /*< ZMPT invert state for PUB/SUB message filters */
+  metadata                  = ZMQ_METADATA, /*< socket metadata key-value pairs */
```

Comment:
I guess I'll have to break down and insert some TAB characters here, much as that makes me cringe...

Reply:
haha, yes... I cringe when I edit a third-party library for things like tabs, too -- or ALL_CAPS CMAKE CALLS